1/hangman.py

```diff
@@ -0,0 +1,45 @@
+# Problem: the game is currently case-sensitive. A != a
+import getpass
+
+# Split the keyword into characters and put them into an array
+def split(keyword):

```

Comment:
usually there's no need to split a string into chars. it's already essentially a array of chars

Reply:
if anything i think `list(keyword)` will return an array of chars